### PR TITLE
editing for issue #353

### PR DESCRIPTION
**Pull request by Gasta88**
Hi,

I took the liberty to go through the Cookbook examples and correct the file handling parts as suggested in issue #353.

### CODE REVIEW

File: wiki/Converting_sequence_files.md [hunk 1]
```diff
@@ -28,14 +28,10 @@ directory):
...
 
-input_handle = open("cor6_6.gb", "rU")
-output_handle = open("cor6_6.fasta", "w")
-
-sequences = SeqIO.parse(input_handle, "genbank")
-count = SeqIO.write(sequences, output_handle, "fasta")
-
-output_handle.close()
-input_handle.close()
+with open("cor6_6.gb", "rU") as input_handle, open("cor6_6.fasta", "w") as output_handle:
```
Comment (peterjc) [inline wiki/Converting_sequence_files.md]: You can use the double with statement from Python 2.7 onwards, which is what Biopython 1.69 will require.

However, this does make the line rather long, so perhaps two nested with statements is better here?

**Comment by peterjc** [comment]
I can't see the problem in the GitHub diff view, but TravisCI is failing due to trailing whitespace:

```
$ if grep -n -r '[[:blank:]]$' wiki/*.md; then echo 'Please remove trailing whitespace.'; false; fi
wiki/Converting_sequence_files.md:34:    
Please remove trailing whitespace.
```

Can you double check your changes to ``wiki/Converting_sequence_files.md``?

**Comment by Gasta88** [comment]
Sorted the nested 'with' statements and hopefully the trailing whitespace.

**Comment by peterjc** [comment]
Thank you!

Were you referring to https://github.com/biopython/biopython/issues/353 here?

**Comment by Gasta88** [comment]
Yes, exactly that issue.

**Comment by peterjc** [comment]
https://github.com/biopython/biopython/issues/353 was specifically on the Entrez examples, but you're right we should be using with statements in general. Thanks for fixing these.

**Pull request merged**